cli/command/cli.go

```diff
@@ -59,6 +60,11 @@ func (cli *DockerCli) HasExperimental() bool {
 	return cli.hasExperimental
 }
 
+// OSType returns the operating system the daemon is running on.
+func (cli *DockerCli) OSType() string {
+	return cli.osType
+}
```

Comment:
I'm now realizing that we never should have put these things on `DockerCli` directly. We should have created a new struct to store experimental and default version. That would make the signature for `isSupported()` and the others a lot cleaner.

I'll handle that in a new PR.